lms/envs/devstack.py

```diff
@@ -412,6 +412,9 @@ def should_show_debug_toolbar(request):
 DCS_SESSION_COOKIE_SAMESITE = 'Lax'
 DCS_SESSION_COOKIE_SAMESITE_FORCE_ALL = True
 
+ENABLE_DEMOGRAPHICS_COLLECTION = True
+ENABLE_DEMOGRAPHICS_MODAL = True
+DEMOGRAPHICS_BASE_URL = "http://localhost:18360"
```

Comment:
I noticed this awhile ago in the feature branch but forgot to make a comment on the official PR. I don't think we will want to commit these here, right?